bin/fuel-convert

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python
+import argparse
+import importlib
+import pkgutil
+
+from fuel import converters
+
+
+if __name__ == "__main__":
+    dataset_options = [
+        m[1] for m in pkgutil.iter_modules(converters.__path__) if not m[2]]
+    parser = argparse.ArgumentParser()
+    parser.add_argument("dataset", help="what dataset to convert",
+                        choices=dataset_options)
+    parser.add_argument("-d", "--directory", help="directory in which input " +
+                        "files reside", type=str, default=None)
+    parser.add_argument("-o", "--output-file", help="where to save the " +
+                        "dataset", type=str, default=None)
+    args = parser.parse_args()
+
+    converter_module = importlib.import_module(
+        'fuel.converters.{}'.format(args.dataset))
```

Comment:
Does this force each converter to be implemented as an independent module? Maybe it would be nicer to assume it is an importable function instead, so that we don't have to put every 4-line convert function in a separate file later on?